### ISSUE TITLE
Error when updating Rstudio

### ISSUE THREAD

**Issue by collioud** [wimpysworld/deb-get#836]
Hello,

I have an issue when updating the package Rstudio.
Thanks for looking into it!
Arnaud

### Expected behavior

Updating the packages when using `deb-get update`

### Actual behavior

Got the message:
 ``` [+] Updating /var/cache/deb-get/rstudio.json
  [!] ERROR! Missing required information of website package rstudio:
URL=
VERSION_PUBLISHED=
```
 
### System information

Running on Ubuntu 20.04.6 LTS

deb-get version 0.3.8.1

Rstudio installed : 2023.03.1-446

**Comment by collioud** [comment]
> Publisher changed the source link, this is what they have on their page `wget https://download2.rstudio.org/server/focal/amd64/rstudio-server-2023.06.0-421-amd64.deb`

Yes, this is the new link for Rstudio Server...

For Rstudio Desktop, the new url is:
https://download1.rstudio.org/electron/focal/amd64/rstudio-2023.06.0-421-amd64.deb

**Comment by philclifford** [comment]
```
sudo cp /etc/deb-get/01-main.d/rstudio /etc/deb-get/99-local.d/
```
which will then override the main repo definition.

(for convenience you might want to own `99-local.d`)

Then tweak and test the logic and when it is correct and tested you could consider creating a PR.

They have changed rstudio-server similarly so it would be good to change that as well.

```
sudo cp /etc/deb-get/01-main.d/rstudio-server /etc/deb-get/99-local.d/
sudo sed -i 's/bionic/focal/g' /etc/deb-get/99-local.d/rstudio*
```

you can check using `deb-get show rstudio rstudio-server` that the correct URLs and versions are found after an update.

**Comment by collioud** [comment]
I did that. See the corresponding PR...
Thanks for your help!

**Comment by philclifford** [comment]
closed by #838

**Comment by philclifford** [comment]
You're very welcome.  Just missed a release but everyone updating will get your fix.  Many thanks for your contribution.
